packages/core/src/bin/commands/start.ts

```diff
@@ -292,11 +299,19 @@ export async function start({
 
   metrics.initializeIndexingMetrics(app);
 
-  if (preCompileResult.result.ordering === "omnichain") {
-    runOmnichain(app);
-  } else {
-    runMultichain(app);
+  switch (preCompileResult.result.ordering) {
+    case "omnichain":
+      runOmnichain(app);
+      break;
+    case "multichain":
+      runMultichain(app);
+      break;
+    case "isolated": {
+      isolatedController(app);
```

Comment:
FWIW, I'd expect this to be named runIsolated and for the file(s) to be in the runtime dir